lib/foreman_maintain/utils/service/remote_db.rb

```diff
@@ -38,6 +38,10 @@ def stop
         [0, db_status.last]
       end
 
+      def restart
+        db_status("It can't be restarted")
```

Comment:
This is ambiguous, and I am trying to think through better wording:

- Must be restarted manually
- Cannot be restarted by #{tool-name}
- Remote databases are not managed by #{tool-name}, they must be restarted by you.

Reply:
Maybe. But it's in line with `enable`/`disable` in this file, and @lpramuk pointed out that having some kind of MSG for `start`/`stop` would also be cool.

Further improvement?